### DISCUSSION TITLE
QC Metrics Issue (Too many cells)

### DISCUSSION THREAD

**Discussion by icinco5** [satijalab/seurat#5976] [Q&A] [answered]
I'm new to Seurat and was given a data set to work with that has already been aligned via Cell Ranger. I'm currently using the PBMC 3K vignette to run my analysis but am running into issues with the QC generation of the violin plots. When generating the violin plots, the figure file is about 0.5 GB and looks oversaturated with cells to the point that you can't see the plots at all. Here is the line of code I'm using to get there:

library(dplyr)
library(Seurat)
library(patchwork)

#Reading in the data frame. You'll want to start from the "multiplex" data after allignment
pbmc.data<- Read10X(data.dir = "./raw_feature_bc_matrix/")

#Create object
pbmc <- CreateSeuratObject(counts = pbmc.data$'Gene Expression')

#Add multi capture 
pbmc[['multicap']]=CreateAssayObject(counts=pbmc.data$'Multiplexing Capture')

#Add a new column to our object to house the MT data
pbmc[["percent.mt"]] <- PercentageFeatureSet(pbmc, pattern = "ND1|ND2|ND3|ND4|ND4L|ND5|ND6|COX1|COX2|COX3|ATP6|ATP8|CYTB")

pbmc[["percent.mt"]] <- PercentageFeatureSet(pbmc, pattern = "^MT-")

VlnPlot(pbmc, features = c("nFeature_RNA", "nCount_RNA", "percent.mt"), ncol = 3)

--

When I open the pbmc object I create, I get the following:
An object of class Seurat 
24049 features across 2936548 samples within 2 assays 
Active assay: RNA (24037 features, 0 variable features)
 1 other assay present: multicap

What's alarming is the number of samples considering that our input for sequencing was 60,000 cells with the hope of 20,000 as an output. My working hypothesis, is that I only have access to the "raw_feature_matrix." I've read that CellRanger outputs a filtered matrix as well but for some reason, the person who ran the alignment didn't have one for me to use. I can work on running on the alignment myself to get the filtered, but it will take several days and I'd rather not spend time and resources doing so unless I need to.

Has anyone else run into this issue before? I'm currently using Seurat 4.0.4

![Screen Shot 2022-05-19 at 10 19 07 AM](https://user-images.githubusercontent.com/105873479/169342917-2c4ecbde-0552-4b90-a6f2-5d8286ca135b.png)

**Comment by timoast** [comment, marked as answer]
There's no need to run cellranger again, you can filter cells from the raw matrix based on total counts. One way to do this is setting the `min.features` parameter in `CreateSeuratObject()`. For example, setting `min.features=300` would probably remove most of the empty droplets.

**Reply by icinco5** [reply]
This worked. Thank you!
